### PR TITLE
Adding unit tests for server/secrets.go

### PR DESCRIPTION
**Pull request by willstudy**
#### What I did
Hello, I write a unit test for `server/secrets.go`.
##### How I did
Vim
##### How to verify it
```
go test -v github.com/kubernetes-incubator/cri-o/server
```
##### Description for the changelog
Adding unit tests for server/secrets.go

### CODE REVIEW

File: server/fixtures/secret/testDataA [hunk 1]
```diff
@@ -0,0 +1 @@
+secretDataA
```
Comment (rhatdan) [inline server/fixtures/secret/testDataA]: Add return

File: server/fixtures/secret/testDataB [hunk 2]
```diff
@@ -0,0 +1 @@
+secretDataB
```
Comment (rhatdan) [inline server/fixtures/secret/testDataB]: add return

File: server/fixtures/secret/testDataA (hunk 1 above)
Comment (mrunalp) [reply server/fixtures/secret/testDataA]: I don't think we need returns for secret data. (Typically I haven't seen it in secrets.)

Comment (willstudy) [reply server/fixtures/secret/testDataA]: I agree with you @mrunalp . If we add the `return`,  this test will be more complicated. As we know,  return is represented `\n` in MacOS/Linux or `\r\n` in Windows. @rhatdan

Comment (rhatdan) [reply server/fixtures/secret/testDataA]: Ok, I did not know.  Live and learn.

**Comment by rhatdan** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM.  I tried to retest the failing e2e_rhel but it didn't kick off.  No details to be found for clues to the issue there.

**Comment by TomSweeneyRedHat** [comment]
Ooops, looks like my page didn't refresh, e2e_rhel looks to be re-testing now for this.

**Comment by runcom** [comment]
LGTM

**Pull request merged**